host/kernel/lts2019-chromium/0001-BACKPORT-lts2019-chromium-v5.4-KVM-x86-emulate-wait-for-SIPI-and-SIPI-VMExit.patch

```diff
@@ -0,0 +1,185 @@
+From b2151dc4620a42691c61e6212762d5941baeddf8 Mon Sep 17 00:00:00 2001
```

Comment:
please change the file name to 0001-BACKPORT-KVM-x86-emulate-wait-for-SIPI-and-SIPI-VMExit.patch